comp/netflow/server/status.go

```diff
@@ -0,0 +1,86 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022-present Datadog, Inc.
+
+package server
+
+import (
+	"embed"
+	"encoding/json"
+	"io"
+	"path"
+
+	htmlTemplate "html/template"
+	textTemplate "text/template"
+
+	"github.com/DataDog/datadog-agent/comp/core/status"
+)
+
+//go:embed status_templates
+var templatesFS embed.FS
+
+// Provider provides the functionality to populate the status output
+type Provider struct{}
+
+// Name returns the name
+func (Provider) Name() string {
+	return "NetFlow"
+}
+
+// Section return the section
+func (Provider) Section() string {
+	return "NetFlow"
+}
+
+// JSON populates the status map
+func (Provider) JSON(_ bool, stats map[string]interface{}) error {
+	status := GetStatus()
```

Comment:
I would rather have any improvements for status provider handles on separate PRs to avoid making this more extensive than it is. I'm trying to focus solely on configuring the status component to render the `agent status`.

Let's make those changes in separate PRs